resources/views/Widgets/Common/ImageBoxWidget.twig

```diff
@@ -21,6 +22,7 @@
 {{ Twig.set("name", "" | json_encode) }}
 {{ Twig.set("url", "" | json_encode) }}
 {{ Twig.set("imageUrl", "" | json_encode) }}
+{{ Twig.set("fallbackImageUrl", "" | json_encode) }}
```

Comment:
Hier ist es dasselbe wie beim BackgroundWidget: Es ist unnötig, die Variable, die sich nicht mehr ändern wird, in das Twig-Template durchzuschleifen und dann per Twig-Builder auszulesen.